Gemfile

```diff
@@ -32,8 +32,10 @@ gem 'faraday'
 gem 'fast_jsonapi'
 gem 'figaro'
 gem 'rubocop-rails', require: false
+gem 'travis'
```

Comment:
Not positive travis or rubocop are required for this project but figured they couldn't hurt.